package.json

```diff
@@ -21,21 +21,21 @@
     "release:version": "./scripts/release/version.js"
   },
   "devDependencies": {
-    "@babel/core": "^7.14.3",
-    "@babel/preset-env": "^7.14.4",
-    "@babel/preset-typescript": "^7.13.0",
+    "@babel/core": "^7.14.6",
+    "@babel/preset-env": "^7.14.7",
+    "@babel/preset-typescript": "^7.14.5",
     "@commitlint/cli": "^12.1.4",
     "@rollup/plugin-node-resolve": "^13.0.0",
     "@rollup/plugin-replace": "^2.4.1",
     "@types/acorn": "^4.0.5",
     "@types/babel__core": "^7.1.14",
     "@types/jest": "^26.0.23",
-    "@types/node": "^15.12.2",
-    "@typescript-eslint/eslint-plugin": "^4.26.1",
-    "@typescript-eslint/parser": "^4.26.1",
+    "@types/node": "^15.12.4",
```

Comment:
Theoretically, we are still using `node@14` but I guess that it doesn't no harm to use the `node@15` types.